omod/src/main/webapp/fragments/program/mchcs/mchcsCarePanel.gsp

```diff
@@ -9,14 +9,22 @@
 	dataPointsLeft << [label: "HIV Status", value: calculations.hivStatus]
 	dataPointsLeft << [label: "HEI Outcome", value: calculations.heioutcomes]
     def dataPointsRight = []
-	dataPointsRight << [label: "PCRs Done: ", value: calculations.pcr]
 
-%>
+    obbList = calculations.obbListView
+    for (testResult in obbList) {
+        orderReason = testResult.orderReason
+        pcrDate = testResult.pcrDate.format("yyyy-MM-dd")
+        testResults = testResult.testResults
+        dataPointsRight << [label: "PRCs Done", value: "Order Reason: $orderReason, Date: $pcrDate, Test:  $testResults"]
```

Comment:
This should read PCR. It would be nice to concatenate the label with age in weeks for the test. This should be a date diff of dob and test date
Remove the order reason for now.

Reply:
You can make sure that the results/tests are ordered in desc. Apply some bit of ui formatting.